src/frontend/Komponenter/Behandling/SettPåVent/LokalkontorOppgavevalg.tsx

```diff
@@ -0,0 +1,100 @@
+    sendteOppgaver,
+    settOppgaverMotLokalkontor,
+    oppgaverMotLokalkontor,
+    erBehandlingPåVent,
+}) => {
+    const tidligereSendteLokalkontorOppgaver = sendteOppgaver.map((oppgave) => {
+        return oppgave.vurderHenvendelseOppgave;
+    });
+
+    const [checkedOppgaver, settCheckedOppgaver] = useState([
+        ...tidligereSendteLokalkontorOppgaver,
+        ...oppgaverMotLokalkontor,
+    ]);
+    const erSendt = (oppgave: VurderHenvendelseOppgavetype): boolean =>
+        tidligereSendteLokalkontorOppgaver.includes(oppgave);
+    const erValgt = (oppgave: VurderHenvendelseOppgavetype): boolean => {
+        return oppgaverMotLokalkontor.includes(oppgave);
+    };
+
+    const lagOppgaveSendtTekst = (oppgave: VurderHenvendelseOppgavetype): string | undefined => {
+        const matchedOppgavestatus = sendteOppgaver.find(
+            (status) => status.vurderHenvendelseOppgave === oppgave
+        );
+
+        if (erValgt(oppgave) && erBehandlingPåVent) {
+            return '(Oppgave er sendt ' + dagensDatoFormatert() + ')';
+        }
+        return matchedOppgavestatus?.datoOpprettet
+            ? ' (Oppgave sendt ' + formaterIsoDato(matchedOppgavestatus?.datoOpprettet) + ')'
+            : undefined;
+    };
+
+    const filtrerTidligereSendteOppgaver = (
+        oppgaver: VurderHenvendelseOppgavetype[]
+    ): VurderHenvendelseOppgavetype[] => {
+        return oppgaver.filter((oppgave) => !tidligereSendteLokalkontorOppgaver.includes(oppgave));
+    };
+
+    return (
+        <div>
+            <CheckboxGroup
+                legend="Send oppgave til lokalkontoret"
+                onChange={(oppgaver) => {
+                    settOppgaverMotLokalkontor(filtrerTidligereSendteOppgaver(oppgaver));
+                    settCheckedOppgaver(oppgaver);
+                }}
+                size="small"
+                value={checkedOppgaver}
+            >
+                {aktuelleOppgaver.map((oppgave) => (
+                    <FlexBox>
+                        <Checkbox
+                            disabled={erSendt(oppgave) || erBehandlingPåVent}
+                            key={oppgave}
+                            value={oppgave}
+                        >
+                            {vurderHenvendelseOppgaveTilTekst[oppgave]}
+                        </Checkbox>
+                        <BodyShort size={'small'}>{lagOppgaveSendtTekst(oppgave)}</BodyShort>
+                    </FlexBox>
+                ))}
+            </CheckboxGroup>
+        </div>
+    );
+};
+
+const vurderHenvendelseOppgaveTilTekst: Record<VurderHenvendelseOppgavetype, string> = {
+    INFORMERE_OM_SØKT_OVERGANGSSTØNAD: 'Beskjed om at vi har fått søknad',
+    INNSTILLING_VEDRØRENDE_UTDANNING: 'Forespørsel om innstilling - utdanning',
+};
+enum VurderHenvendelseOppgavetype {
+    INFORMERE_OM_SØKT_OVERGANGSSTØNAD = 'INFORMERE_OM_SØKT_OVERGANGSSTØNAD',
+    INNSTILLING_VEDRØRENDE_UTDANNING = 'INNSTILLING_VEDRØRENDE_UTDANNING',
+}
```

Comment:
Du kunne vurdert å lage en egen mappe for `LokalkontorOppgavevalg` med en egen `typer.ts`-fil. Tenker i hvert fall at denne kan flyttes til toppen av fila der vi har de andre typedefinisjonene